clarity/src/vm/contexts.rs

```diff
@@ -822,6 +822,16 @@ impl<'a, 'hooks> OwnedEnvironment<'a, 'hooks> {
             self.context.eval_hooks = Some(vec![hook]);
         }
     }
+
+    #[cfg(any(test, feature = "testing"))]
```

Comment:
Nit: This should go into the test module

Reply:
Addressed in 7a63f899e8945a770642517e96324330597dbe87.

Reply:
Turns out moving this into the test module creates some difficulty elsewhere because of our use of `#[cfg(any(test, feature = "testing"))]` in various places. I pulled it out of this impl block and put it in its own block with `#[cfg(any(test, feature = "testing"))]` instead.